src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,42 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+
+type Props = {
+  todo?: Todo | null,
```

Comment:
<img width="557" alt="image" src="https://github.com/mate-academy/react_dynamic-list-of-todos/assets/70220667/c13518e9-b784-4c9d-955a-636945f17335">
```suggestion
  todo: Todo,
```
looks like you render this component only when todo exists